content/administration/install/install.rst

```diff
@@ -270,31 +272,42 @@ Enterprise Edition:
 * `Official download page <download_>`_
 * `GitHub repository <enterprise-repository_>`_
 
+.. _setup/install/source/windows/git:
+
 Git
 ^^^
 
-The following requires git_ to be installed on your machine and that you have basic knowledge of
-git commands.
+The following requires `Git <git_>`_ to be installed on your machine and that you have basic
+knowledge of Git commands. To clone a Git repository, you must choose between cloning with HTTPS or
+SSH. If you do not know the difference between the two, the best option is most likely HTTPS. If you
+are following the :doc:`Getting started </developer/howtos/rdtraining>` developer tutorial, or plan
+on contributing to Odoo source code, choose SSH.
```

Comment:
The entire page is duplicated, not only these notes. I could use shared RST files to address that but, given the number of duplicated paragraphs, it could clutter the page more than it cleans it. In later versions I refactored the page to use [content tabs](https://www.odoo.com/documentation/master/contributing/documentation/rst_cheat_sheet.html#content-tabs) instead of one separate section per OS; this allowed me to get rid of the duplications. So I will just merge as-is in 14.0 and remove the duplicates in the forward-ports. ;)